.github/workflows/image-master.yaml

```diff
@@ -220,11 +212,17 @@ jobs:
       matrix:
         base_image:
           - "ubuntu:24.04"
-          - "ghcr.io/kairos-io/hadron:v0.0.1-beta1"
         test:
           - "provider-qrcode-install"
           - "provider-upgrade"
           - "provider-upgrade-k8s"
           - "provider-upgrade-latest-k8s-with-kubernetes"
           - "provider-decentralized-k8s"
           - "provider-k3s-disabled"
+        include:
+          - base_image: "ghcr.io/kairos-io/hadron:v0.0.1-beta2"
```

Comment:
I wonder if we can somehow set this once by using github action env variables. Alternatively, we can set this somewhere on the repository, since all actions checkout the code anyway.